wrappers/angular-handsontable/.travis.yml

```diff
@@ -0,0 +1,17 @@
+dist: xenial
```

Comment:
Correct me if I'm wrong. But this file can be deleted. Tests run on the CodeShip, now?

Reply:
I believe so 👍